cmk.go

```diff
@@ -70,6 +82,10 @@ func main() {
 	}
 
 	if *profile != "" {
+		if !existsProfileCache(*profile, cfg) {
+			fmt.Printf("Cannot find a cache file for the profile: %s\n", *profile)
+			os.Exit(1)
```

Comment:
I’m not sure if we should exit if cache file isn’t available?

Reply:
I would have thought of it using a cache file otherwise giving a warning to the user when not found, and when it’s found just use it.

Reply:
agree with the approach

Reply:
Thanks, fixing it